### PR TITLE
Quality: PowerShell Summit GH-3408

### LINKED ISSUE #3408: Quality: Uninstall-ADDSDomainController.md
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Aliases, Formatting code samples, Formatting command syntax, Link References, Markdown linting, Spelling

### Article List

- Uninstall-ADDSDomainController.md

### PR DESCRIPTION
**Pull request by RobBiddle**
# PR Summary

- Resolves #3408 
## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md [hunk 1]
```diff
@@ -62,12 +71,14 @@ Accept wildcard characters: False
...
 
 ### -Credential
-Specifies the user name and password that corresponds to the account used to install the domain controller.
-Use the **Get-Credential** cmdlet to prompt the user to supply a password in place of an existing *PSCredential* type.
-This causes Windows PowerShell to prompt the user to enter credentials using the Windows security login UI.
+
+Specifies the user name and password that corresponds to the account used to install the domain
+controller. Use the `Get-Credential` cmdlet to prompt the user to supply a password in place of an
+existing _System.Management.Automation.PSCredential_ type. This causes Windows PowerShell to prompt
+the user to enter credentials using the Windows security login UI.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md:78]:
```suggestion
Specifies the user name and password that corresponds to the account used to install the domain
controller. Use the `Get-Credential` cmdlet to prompt the user to supply a password in place of an
existing **System.Management.Automation.PSCredential** type. This causes Windows PowerShell to
prompt the user to enter credentials using the Windows security login UI.
```

File: docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md [hunk 2]
```diff
@@ -94,12 +107,14 @@ Accept wildcard characters: False
...
 
 ### -DnsDelegationRemovalCredential
-Specifies the account credentials to use when you create or remove the DNS delegation.
-If you do not specify a value, the account credentials that you specify for the AD DS installation or removal are used to remove the DNS delegation.
-As an alternative, you can specify the asterisk (*) to prompt the user to enter credentials.
+
+Specifies the account credentials to use when you create or remove the DNS delegation. If you do not
+specify a value, the account credentials that you specify for the AD DS installation or removal are
+used to remove the DNS delegation. As an alternative, you can specify the asterisk (*) to prompt the
+user to enter credentials.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md:114]:
```suggestion
Specifies the account credentials to use when you create or remove the DNS delegation. If you do
not specify a value, the account credentials that you specify for the AD DS installation or removal
are used to remove the DNS delegation. As an alternative, you can specify the asterisk (`*`) to
prompt the user to enter credentials.
```

File: docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md [hunk 3]
```diff
@@ -142,12 +160,19 @@ Accept wildcard characters: False
...
+
+Indicates that Windows PowerShell ignores any inconsistency that it detects with the value that you
+specify for the `-LastDomainControllerInDomain` parameter. For instance, if you specify
+`-LastDomainControllerInDomain` but Windows PowerShell detects that there is actually another active
+domain controller in the domain, you can specify the `-IgnoreLastDCInDomainMismatch` parameter to
+have Windows PowerShell continue the removal of AD DS from the domain controller despite the
+inconsistency that it has detected. Similarly, if you do not specify `-LastDomainControllerInDomain`
+but Windows PowerShell cannot detect that another domain controller is in the domain, you can
+specify `-IgnoreLastDCInDomainMismatch` to have Windows PowerShell continue to remove AD DS from the
+domain controller.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md:172]:
```suggestion
Indicates that Windows PowerShell ignores any inconsistency that it detects with the value that you
specify for the **LastDomainControllerInDomain** parameter. For instance, if you specify
**LastDomainControllerInDomain** but Windows PowerShell detects that there is actually another
active domain controller in the domain, you can specify the **IgnoreLastDCInDomainMismatch**
parameter to have Windows PowerShell continue the removal of AD DS from the domain controller
despite the inconsistency that it has detected. Similarly, if you do not specify
**LastDomainControllerInDomain** but Windows PowerShell cannot detect that another domain
controller is in the domain, you can specify **IgnoreLastDCInDomainMismatch** to have Windows
PowerShell continue to remove AD DS from the domain controller.
```

File: docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md [hunk 4]
```diff
@@ -278,13 +320,17 @@ Accept wildcard characters: False
...
-When this switch parameter is set, it specifies that additional preliminary checks should be bypassed.
-For more information on the scope of these additional preliminary checks that the ADDSDeployment module performs by default when using Windows Server 2012, refer to the table in the section ADPrep and Prerequisite Checking Architecture in [AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
+
+Indicates that only a base set of validations is performed. This behavior is equivalent to the
+validations that were performed when using `Dcpromo.exe` in earlier versions of Windows Server to
+add a new domain controller. When this switch parameter is set, it specifies that additional
+preliminary checks should be bypassed. For more information on the scope of these additional
+preliminary checks that the ADDSDeployment module performs by default when using Windows Server
+2012, refer to the table in the section ADPrep and Prerequisite Checking Architecture in
+[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md:330]:
```suggestion
Indicates that only a base set of validations is performed. This behavior is equivalent to the
validations that were performed when using `Dcpromo.exe` in earlier versions of Windows Server to
add a new domain controller. When this switch parameter is set, it specifies that additional
preliminary checks should be bypassed. For more information on the scope of these additional
preliminary checks that the **ADDSDeployment** module performs by default when using Windows Server
2012, refer to the table in the section ADPrep and Prerequisite Checking Architecture in
[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```

File: docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md [hunk 5]
```diff
@@ -79,10 +90,12 @@ Accept wildcard characters: False
 ```
 
 ### -DemoteOperationMasterRole
-Indicates that forced demotion should continue even if an operations master role is discovered on domain controller from which AD DS is being removed.
+
+Indicates that forced demotion should continue even if an operations master role is discovered on
+domain controller from which AD DS is being removed.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md:95]:
```suggestion
the domain controller from which AD DS is being removed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RobBiddle, one minor additional change, please. Many thanks.

File: docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md [hunk 6]
```diff
@@ -11,46 +11,55 @@ title: Uninstall-ADDSDomainController
 # Uninstall-ADDSDomainController
 
 ## SYNOPSIS
+
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/addsdeployment/Uninstall-ADDSDomainController.md:14]:
```suggestion
```

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**